builtin/logical/ssh/storage.go

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) 2024 OpenBao a Series of LF Projects, LLC
+// SPDX-License-Identifier: MPL-2.0
+
+package ssh
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openbao/openbao/sdk/v2/helper/errutil"
+	"github.com/openbao/openbao/sdk/v2/logical"
+)
+
+const (
+	defaultRef = "default"
+
+	issuerPrefix        = "config/issuer/"
+	storageIssuerConfig = "config/issuers"
+
+	caPublicKey                       = "ca_public_key"
+	caPrivateKey                      = "ca_private_key"
+	caPublicKeyStoragePath            = "config/ca_public_key"
+	caPublicKeyStoragePathDeprecated  = "public_key"
+	caPrivateKeyStoragePath           = "config/ca_private_key"
+	caPrivateKeyStoragePathDeprecated = "config/ca_bundle"
+
+	// NOTE (gabrielopesantos): These might be to high for SSH issuers
+	maxRolesToScanOnIssuerChange = 100
+	maxRolesToFindOnIssuerChange = 10
+
+	// Used as a quick sanity check for a reference id lookups...
+	uuidLength = 36
+)
+
+type issuerID string
```

Comment:
This is nice in the PKI land, so to not confuse `keyID` vs `issuerID` but may not be necessary here. It added a bit of problems in PKI in particular, though I think one of those problems will be fixed by #887 tbh (typing).

Reply:
That sounds good, I defined this type to stay consistent with PKI but don't seem much use in having it, going to revert.